be/src/storage/rowset/rowset.cpp

```diff
@@ -294,7 +294,7 @@ Status Rowset::_remove_delta_column_group_files(std::shared_ptr<FileSystem> fs)
             for (const auto& dcg : list) {
                 auto st = fs->delete_file(dcg->column_file());
                 if (st.ok() || st.is_not_found()) {
-                    LOG(INFO) << "Deleting delta column group's file: " << dcg->debug_string() << " st: " << st;
+                    VLOG(1) << "Deleting delta column group's file: " << dcg->debug_string() << " st: " << st;
```

Comment:
Add the filename to the log

Reply:
The filename is included in `debug_string()`